library-definitions/proguard-rules.txt

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```

Comment:
I would remove this file and reference to it in build.gradle if does not have any used content.